spec/requests/vendor_api/v1.1/get_single_application_with_obfuscated_references_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'rails_helper'
```

Comment:
is `obfuscated` still the appropriate word here (in the filename)?

Reply:
🙅